android/expoview/src/main/java/host/exp/exponent/notifications/channels/ScopedNotificationsChannelManager.java

```diff
@@ -0,0 +1,70 @@
+package host.exp.exponent.notifications.channels;
+
+import android.app.NotificationChannel;
+import android.content.Context;
+import android.os.Build;
+
+import org.unimodules.core.arguments.ReadableArguments;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import expo.modules.notifications.notifications.channels.managers.AndroidXNotificationsChannelManager;
+import expo.modules.notifications.notifications.channels.managers.NotificationsChannelGroupManager;
+import host.exp.exponent.kernel.ExperienceId;
+
+public class ScopedNotificationsChannelManager extends AndroidXNotificationsChannelManager {
+
+  private ExperienceId mExperienceId;
+
+  public ScopedNotificationsChannelManager(Context context, ExperienceId experienceId, NotificationsChannelGroupManager groupManager) {
```

Comment:
I don't think this is a good place for composition. Substituting inheritance with composition will generate more code. Moreover, generated abstraction will be confusing for users who can contribute to our repo. 
For example, to refactor `ScopedGroupSerializer`, I need to create abstraction which will get `NotificationGroup` and returns `NotificationGroup.getId()`. For someone who wants to change something in `expo-notifications`, this can be very confusing and unnecessary.  Moreover, rewriting `ScopedNotificationsChannelManager` will be complicated, cause I sometimes need to fall back to the non-scoped method.